validator/src/hashlib.rs

```diff
@@ -0,0 +1,24 @@
+use hex;
+use openssl;
+
+pub fn sha256_digest_str(item: &str) -> String {
+    let mut bytes: Vec<u8> = Vec::new();
```

Comment:
Would it be easier just do something like
```
let hash = sha256(foo);
    hex::encode(hash)
```

Reply:
Yep, added a commit